src/main/resources/static/index.html

```diff
@@ -95,48 +95,58 @@ <h3 class="ui left floated huge red header">
-                    ponderum tractatos ei.</p>
-                    <div class="ui container">
-                      <a href="/signup.html">
-                        <button class="ui large red button" id="Register">Register now</button>
-                      </a>
+                <div class="mainContainer" id="mainContainer" style="display: block">
+                    <div class="ui inverted blue segment" id="logoutLabel" style="display: none">
+                        <p>you have successfully been logged out</p>
                     </div>
-              </div>
-
-              <div class="loginContainer" id="loginContainer" style="display: none">
-                <div class="ui placeholder segment" id="fogCover" style="background: rgba(0,0,0,.7);">
-                  <h2>Login</h2>
-                  <form class="ui form" method="post" action="/">
-                      <label style="text-align: left;">Email</label>
-                      <input type="text"  name="username" placeholder="email" />
-                      <label style="text-align: left;">Password</label>
-                      <input type="password" placeholder="password" name="password" />
-                      <input type="submit"/>
-                  </form>
-                  <div class="ui inverted green segment" id="confirm" style="display: none">
-                    <p>you have successfully been logged in</p>
-                  </div>
-                  <div class="ui inverted red segment" id="deny">
-                    <p>Wrong Email or password</p>
-                  </div>
-                  <hr>
-                  <div class="ui inverted header">Don't have an account?<br>
-                      <a href="/signup.html">
-                          <div class="ui big button">
-                              <i class="signup icon"></i>
-                              Sign Up
-                          </div>
+                    <h1 style="font-size:4vw;">Donate Blood To Save Life</h1>
+
+                    <iframe width="728" height="409" src="https://www.youtube.com/embed/qEo2wrYyCRI"
+                            frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope;
+                          picture-in-picture" allowfullscreen>
+                    </iframe>
+                    <br>
+                    **click the "YouTube" link in the video player for higher resolution video**
+
+                    <br>
+                    <br>
+                    <div class="ui container">
+                        <a href="/signup.html">
+                            <button class="ui large red button" id="Register">Register now</button>
                         </a>
-                  </div>
+                        <button class="ui large basic inverted button" id="Search" onclick="tempButton()">Search for
+                            blood
+                        </button>
```

Comment:
Please remove this button as it doesn't actually work.

Reply:
Weird.. did I somehow add that? No matter  - it's gone